src/dmd/ob.d

```diff
@@ -287,7 +287,7 @@ struct PtrVarState
         assert(vars.length == deps.length);
         OutBuffer buf;
         depsToBuf(buf, vars);
-        string t = buf[];
+        auto t = buf[];
```

Comment:
?

Reply:
For some reason the code refused to compile without this change.

Reply:
Hmm, fair. I guess the data is const or mutable then?